src/components/ec-alert/ec-alert.vue

```diff
@@ -153,54 +174,68 @@ export default {
     @apply tw-rounded;
     @apply tw-absolute tw-top-8 tw-right-8;
     @apply tw-cursor-pointer;
-    @apply tw-fill-gray-8;
 
     line-height: 0;
 
     &:focus {
       @apply tw-outline-none;
-      @apply tw-bg-gray-8;
-    }
-
-    &:hover {
-      @apply tw-bg-gray-8;
     }
 
     &--info:focus,
     &--info:hover {
       @apply tw-fill-info;
     }
 
+    &--info {
```

Comment:
specificity takes care of this (I think) but IMO we should have this prior to the psuedoclasses avobe and nest

```
    &--info {
      ...
      &:focus,
      &:hover {
      @apply tw-fill-info;
    }
```

Reply:
You're right! I've just changed it! Thanks!